modules/ROOT/partials/configuration/exportword.adoc

```diff
@@ -0,0 +1,27 @@
+[[exportword-service-url]]
+== `exportword_service_url`
+
+By default, the Word export feature uses the {productname} Cloud Services HTML to DOCX converter service to generate the Word files. You can use this option to provide the URL to an on-premises converter.
+
+*Type:* `+String+`
+
+=== Example: using `exportword_service_url`
+
+// Add a working and tested configuration.
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'code exportword',
+  toolbar: 'exportword code',
+  exportword_service_url: '<value>'
```

Comment:
@hamza0867 Is this the best way to present `exportword_service_url:` in editor config?